ansible-ipi-install/roles/installer/templates/metal3-config.j2

```diff
@@ -13,5 +13,5 @@ data:
   ironic_inspector_endpoint: http://{{ prov_ip }}:5050/v1/
   provisioning_interface: {{ prov_nic }}
   provisioning_ip: {{ prov_ip }}/24
-  rhcos_image_url: {{ rhcos_path }}{{ rhcos_uri }}
+  rhcos_image_url: {{ rhcos_path }}{{ rhcos_openstack_uri }}
```

Comment:
do we want to have the word openstack in here?

Reply:
The images downloaded are in fact the `openstack` ones, i.e. `rhcos-44.81.202001241431.0-openstack.x86_64.qcow2.gz`.  Would could call it something else though.

Reply:
id almost prefer to call it rhcos_uri and not have anything relating to osp. For all we know they will change the image to something else but we wouldn't need to worry since ours is rhcos_uri versus calling out osp in the name.

Reply:
Understood.  Another thing I forgot to mention was that I used `openstack` to differentiate from the bootstrap image, which is `qemu`.  How about we keep `rhcos_qemu_uri` and leave the other one as it was before this PR: `rhcos_uri`?